frontend/src/component/admin/groups/GroupsList/GroupCard/GroupCard.tsx

```diff
@@ -101,6 +113,28 @@ export const GroupCard = ({
                             />
                         </StyledHeaderActions>
                     </StyledTitleRow>
+                    <ConditionallyRender
+                        condition={!!group.rootRole}
+                        show={
+                            <>
```

Comment:
No need for fragment `<>` since it seems like you only have 1 child in this case: `<InfoBadgeDescription>`

Reply:
Oh my word, that finally makes sense to me. Thank you!